browser/ftx/ftx_protocol_handler.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_FTX_FTX_PROTOCOL_HANDLER_H_
+#define BRAVE_BROWSER_FTX_FTX_PROTOCOL_HANDLER_H_
+
+#include <string>
+
+#include "chrome/browser/external_protocol/external_protocol_handler.h"
```

Comment:
This header is not used here and you are missing headers for GURL, WebContents Optional and Origin. See https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md

Reply:
fixed via https://github.com/brave/brave-core/compare/6f4c503619e82a440ef7db9782288e922c37d5e4..f4d887c0baf39543331107f2e22f9b0ad4986c02